# generated by patch-package 6.4.8 on 2022-01-12 10:55:43
#
# command:
#   npx patch-package rollup-plugin-workers
#
# declared package:
#   rollup-plugin-workers: 2.1.0
#
diff --git a/node_modules/rollup-plugin-workers/index.js b/node_modules/rollup-plugin-workers/index.js
index 4434cc1..84efa97 100644
--- a/node_modules/rollup-plugin-workers/index.js
+++ b/node_modules/rollup-plugin-workers/index.js
@@ -68,7 +68,7 @@ function findWorklet(node) {
   const { callee } = callExpression;
   if (callee.type !== 'MemberExpression') return;
   if (callee.object.type !== 'MemberExpression') return;
-  if (!callee.object.property.name.match(/^[a-z]+Worklet$/)) return;
+  if (!callee?.object?.property?.name?.match(/^[a-z]+Worklet$/)) return;
   if (callee.property.name !== 'addModule') return;
 
   return callExpression;
